datamade/ccfp-asset-dashboard
# Add phase and budget to the project form

## Overview
- added phase and estimated_bid_quarter as fields on the Project model
- added phase, estimated_bid_quarter, and budget to the project detail form
- tests

### Demo
![phase-budget-demo](https://user-images.githubusercontent.com/38969506/107276971-69ff5580-6a19-11eb-9443-3ab67e6aa08e.gif)

## Testing Instructions
- visit the review app
- add a project
- update the new fields
- confirm that the data persists upon reload
